content/administration/upgrade.rst

```diff
@@ -1,12 +1,252 @@
+
+.. _upgrade/test-guidance:
+
+Test guidance
+=============
+
+Every business and organization has its own operational needs and has to test its specific Odoo
+database individually. We recommend you look at `the test scenario
+<https://docs.google.com/document/d/1ypNs7JKPOsjNbKpdiKFH7Al6g6whZ9jr7f7duAQ5E1w/>`_ for further
+information.
+
+.. todo:: change link "test scenario" once the related doc is published
+
+.. _upgrade/test-assistance:
+
+Assistance
+----------
+
+If you encounter an issue in the **test database**, please get in touch with Odoo Upgrade Support
+via the `Odoo Support page <https://www.odoo.com/help>`_.
+
+Under the *Ticket Description* section, select *An issue related to my upgrade* ticket type.
+
+   .. image:: upgrade/test-assistance.png
+      :align: center
+      :alt: Selection of "An issue related to my upgrade" as Ticket Type in the support form on Odoo
+
+   .. warning::
+      If you choose another *Ticket Description* type, the request will be redirected to another
+      team. This will slow down the processing and response time.
+
+Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
+This will avoid clarifying questions and speed up the resolution process significantly.
+
+.. note::
+   * The purpose of the test phase is not to correct existing data or configurations in your
+     database.
+   * |assistance-contact|
+
+.. _upgrade/steps-production:
+
+The production launch
+=====================
+
+The production upgrade request is when you decide to upgrade your current database with all your
+production data (invoices, VAT returns, inventories, current orders) to a new version of your
+choice.
+
+After your :ref:`tests <upgrade/testing-phase>` are completed to your satisfaction, submit the
+request to upgrade your production database via our `website form <https://upgrade.odoo.com>`_.
+Select *Production* purpose.
+
+.. important::
+   Going into production without first testing may lead to:
+
+   - business interruptions (e.g., no longer having the possibility to validate an action)
+   - poor customer experiences (e.g., an eCommerce website that does not work correctly)
+
+.. _upgrade/production-assistance:
+
+Assistance
+----------
+
+If you encounter issues or problems in the **production database**, please get in touch with **Odoo
+Support**:
+
+#. Connect to our `Odoo Support page <https://www.odoo.com/help>`_.
+#. Under the *Ticket Description* section, select the appropriate type related to your issue but
+   **do not select** the option *An issue related to my upgrade*.
+
+   .. note::
+      After upgrading to production, the support will be provided by the Support team instead of the
+      Upgrade team.
+
+#. Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
+   This will avoid clarifying questions and speed up the resolution process significantly.
```

Comment:
Please refer to the line before. We mean here that if they provide as much details as possible, it will then avoid (and not allow) to have to clarify questions later.